packages/serverless-workflow-diagram-editor/sw-editor/sw-editor-kogito-app/src/main/java/org/kie/workbench/common/stunner/sw/client/editor/ShapeEventsHandler.java

```diff
@@ -17,26 +17,29 @@
 package org.kie.workbench.common.stunner.sw.client.editor;
 
 import javax.enterprise.event.Observes;
+import javax.inject.Inject;
 import javax.inject.Singleton;
 
-import elemental2.dom.DomGlobal;
+import org.appformer.kogito.bridge.client.diagramApi.DiagramApi;
 import org.kie.workbench.common.stunner.core.client.canvas.CanvasHandler;
-import org.kie.workbench.common.stunner.core.client.canvas.event.selection.CanvasClearSelectionEvent;
 import org.kie.workbench.common.stunner.core.client.canvas.event.selection.CanvasSelectionEvent;
 import org.kie.workbench.common.stunner.core.graph.Node;
 import org.kie.workbench.common.stunner.core.graph.content.view.View;
 import org.kie.workbench.common.stunner.sw.definition.State;
 
+
 @Singleton
 public class ShapeEventsHandler {
 
+    @Inject
+    private DiagramApi DiagramApi;
```

Comment:
```suggestion
    private DiagramApi diagramApi;
```
Please start variable names in lowercase.